drivers/homeless_summary_report/app/models/homeless_summary_report/client.rb

```diff
@@ -0,0 +1,41 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HomelessSummaryReport
+  class Client < GrdaWarehouseBase
+    acts_as_paranoid
+
+    has_many :simple_reports_universe_members, inverse_of: :universe_membership, class_name: 'SimpleReports::UniverseMember', foreign_key: :universe_membership_id
+    belongs_to :report
+
+    # Create a scope for each report variant
+    Report.report_variants.each_key do |variant|
+      scope variant, -> { where(arel_table["spm_#{variant}".to_sym].gt(0)) }
+    end
+
+    scope :spm_m1a_es_sh_days, -> { where(arel_table[:spm_m1a_es_sh_days].gt(0)) }
+    scope :spm_m1a_es_sh_th_days, -> { where(arel_table[:spm_m1a_es_sh_th_days].gt(0)) }
+    scope :spm_m1b_es_sh_ph_days, -> { where(arel_table[:spm_m1b_es_sh_ph_days].gt(0)) }
+    scope :spm_m1b_es_sh_th_ph_days, -> { where(arel_table[:spm_m1b_es_sh_th_ph_days].gt(0)) }
+
+    # TODO: Is this the right query?
```

Comment:
I believe it is.  SPM M2 buckets like so:
```
reentered_0_5 = scope.where(t[:m2_reentry_days].between(1 .. 180))
reentered_6_12 = scope.where(t[:m2_reentry_days].between(181 .. 365))
reentered_13_24 = scope.where(t[:m2_reentry_days].between(366 .. 730))
reentered = scope.where(t[:m2_reentry_days].between(1 .. 730))
```
Which would indicate we're using 0 as not returning.